ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -134,7 +136,7 @@ tests:
     env:
       OO_CHANNEL: alpha
       OO_INSTALL_NAMESPACE: file-integrity
-      OO_LATEST_CSV: file-integrity-operator.v1.3.0
+      OO_LATEST_CSV: file-integrity-operator.v1.3.3
```

Comment:
This should probably be `file-integrity-operator.v1.3.4` based on the documentation:

_OO_LATEST_CSV 	The name of the CSV being upgraded to._

Reply:
And I wonder why is `INITIAL_CSV` is not set.

````
INITIAL_CSV 	Parameter[[?](https://docs.ci.openshift.org/docs/architecture/step-registry/#parameters)] 
(For upgrade tests) The name of the initial CSV to install.
```